HW3/classifier.py

```diff
@@ -19,7 +19,7 @@ def train_classifier(features, labels, args):
 
 def make_predictions(clf, features):
     prediction_probs = clf.predict_proba(features)
-    return np.argmax(prediction_probs, axis=-1), \
+    return np.argmax(prediction_probs, axis=-1)+1, \
```

Comment:
Thank you for the fix. Can you put an space before and after the `+` operator to follow the code formatting?

Reply:
Sure, one sec